client/app/telecom/telephony/alias/configuration/mode/easyHunting/telecom-telephony-alias-configuration-mode-easyHunting.html

```diff
@@ -26,17 +28,8 @@ <h4 data-translate="loading"></h4>
                 </header>
                 <p data-translate="telephony_alias_configuration_mode_easyhunting_info1"></p>
                 <ul class="ml-3">
-                    <li data-translate="telephony_alias_configuration_mode_easyhunting_info2"></li>
-                    <li data-translate="telephony_alias_configuration_mode_easyhunting_info3"></li>
-                    <li data-translate="telephony_alias_configuration_mode_easyhunting_info4"></li>
-                    <li data-translate="telephony_alias_configuration_mode_easyhunting_info5"></li>
-                    <li data-translate="telephony_alias_configuration_mode_easyhunting_info6"></li>
-                    <li data-translate="telephony_alias_configuration_mode_easyhunting_info7"></li>
-                    <li data-translate="telephony_alias_configuration_mode_easyhunting_info8"></li>
-                    <li data-translate="telephony_alias_configuration_mode_easyhunting_info9"></li>
-                    <li data-translate="telephony_alias_configuration_mode_easyhunting_info10"></li>
-                    <li data-translate="telephony_alias_configuration_mode_easyhunting_info11"></li>
-                    <li data-translate="telephony_alias_configuration_mode_easyhunting_info12"></li>
+                    <li data-ng-repeat="i in [].constructor(11) track by $index"
```

Comment:
It's good to refactor but I would prefer to set index into the array :
```[2,3,4,5,6,7,8,9,10,11,12]```
and then use ```i``` to build translation key